standalone/install.ts

```diff
@@ -1,7 +1,7 @@
 import chalk = require('chalk');
 
 // Get latest version from https://github.com/pact-foundation/pact-ruby-standalone/releases
-export const PACT_STANDALONE_VERSION = '1.92.0';
+export const PACT_STANDALONE_VERSION = '2.2.1';
```

Comment:
🙌

Reply:
2.0.0 exists in pact-foundation/pact-ruby-standalone with ruby 3.2.2

but issues in the pact mock service in 3.2.2 caused us to need to roll back to 3.1.2

thankfully i’d done my travelling ruby upgrades 3.1 and 3.2 so had 3.1 version of travelling ruby

Reply:
now using 2.0.2 latest from pact-ruby-standalone